fvm/derived/table.go

```diff
@@ -424,6 +426,13 @@ func (txn *TableTransaction[TKey, TVal]) GetOrCompute(
 
 	val, err = computer.Compute(txnState, key)
 	if err != nil {
+		// drop the nested transaction in case the caller wants to
```

Comment:
Dropping the nested transaction will result in incorrect spock hash computation?  The Compute call accessed a bunch of registers.  but dropping the nested transaction, you're basically saying that those register access never happened.

you probably want to restart then commit the nested transaction instead of dropping it.

Reply:
hmm ... 

that is a great point. Committing would be better. I'll change it.